DeAceroDLabs/marketplace-frontend
# [DMP-719] Display Offers

## What is done

[[DMP-719] Mostrar ofertas](https://deacero.atlassian.net/browse/DMP-719)

## How is done

1. Added a new route on API to point to offers.
2. Modified the OrderOptions component to use the real offers.
3. Modified the Offers to use real offers.
4. Added a new function to get estimated dates.
5. Modified styles on offers and offers cards.

## How to test

1. Add products to your shopping-cart. Here I have a list of items that work:
<img width="780" alt="Screenshot 2022-11-28 at 8 37 54" src="https://user-images.githubusercontent.com/68872548/204318213-2800555a-39d8-4eae-881f-9e228ded39c8.png">

2. Once you successfully added the products, go to the carrito page.
3. Then click on "Ir a pagar"
4. You should see the offers there


## Evidence

Add images or screen  recordings on how the feature was working after and before the fix (if applies)
